sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_async.py

```diff
@@ -802,3 +804,100 @@ async def test_multiple_of_same_action_fail(self, client):
+            "Brexit negotiations, David Davis. Their decision to leave the government came three days after May "
+            "appeared to have agreed a deal with herfractured Cabinet on the UK's post Brexit relationship with "
+            "the EU. That plan is now in tatters and her political future appears uncertain. May appeared in Parliament"
+            " on Monday afternoon to defend her plan, minutes after Downing Street confirmed the departure of Johnson. "
+            "May acknowledged the splits in her statement to MPs, saying of the ministers who quit: We do not agree "
+            "about the best way of delivering our shared commitment to honoring the result of the referendum. The "
+            "Prime Minister's latest plitical drama began late on Sunday night when Davis quit, declaring he could "
+            "not support May's Brexit plan. He said it involved too close a relationship with the EU and gave only "
+            "an illusion of control being returned to the UK after it left the EU. It seems to me we're giving too "
+            "much away, too easily, and that's a dangerous strategy at this time, Davis said in a BBC radio "
+            "interview Monday morning. Johnson's resignation came Monday afternoon local time, just before the "
+            "Prime Minister was due to make a scheduled statement in Parliament. This afternoon, the Prime Minister "
+            "accepted the resignation of Boris Johnson as Foreign Secretary, a statement from Downing Street said."},
+            {"id": "2", "language": "es", "text": "Microsoft fue fundado por Bill Gates y Paul Allen"}]
+
+        async with client:
+            response = await (await client.begin_analyze_actions(
+                docs,
+                actions=[ExtractSummaryAction()],
+                show_stats=True,
+                polling_interval=self._interval(),
+            )).result()
+
+            document_results = []
+            async for doc in response:
+                document_results.append(doc)
+
+            assert len(document_results) == 2
+            for document_result in document_results:
+                assert len(document_result) == 1
+                for result in document_result:
+                    assert isinstance(result, ExtractSummaryResult)
+                    assert result.statistics
+                    assert len(result.sentences) == 3 if result.id == 0 else 1
+                    for sentence in result.sentences:
+                        assert sentence.text
+                        assert sentence.rank_score is not None
+                        assert sentence.offset is not None
+                        assert sentence.length is not None
+                    assert result.id is not None
+
+    @TextAnalyticsPreparer()
+    @TextAnalyticsClientPreparer()
+    async def test_extract_summary_action_with_options(self, client):
```

Comment:
is there a way to add a test where this action fails? just to verify that the proper parsing is working

Reply:
done